crypto/asn1/a_time.c

```diff
@@ -130,7 +130,7 @@ int ASN1_TIME_set_string(ASN1_TIME *s, const char *str)
     return 1;
 }
 
-static int asn1_time_to_tm(struct tm *tm, const ASN1_TIME *t)
+int ASN1_TIME_to_tm(struct tm *tm, const ASN1_TIME *t)
```

Comment:
Yup; I just un-static-fied the exiting function, but I agree, the arguments should be reversed.